Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
We disagree that it's important because it does not catch bugs in our experiences. Our `test do` blocks are better tests.